README.rst

```diff
@@ -12,6 +12,11 @@ Simple `Let’s Encrypt`_ client.
       -d example.com -d www.example.com --default_root /var/www/html \
       -d example.net:/var/www/other_html
 
+This creates a new key file and requests a single certificate for
+``example.com`` and ``example.net``. The key files and certificates are
+stored in the current working directory. If no ``account_key.json`` file
+is found, a new account is registered using ``you@example.com``.
```

Comment:
I personally find this paragraph a bit misleading as it doesn't show the true nature of `simp_le`: i.e. the idempotent behavior (creationg of new files only if necessary, etc.).